draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1463,15 +1450,36 @@ Steps:
 22. return (x, y)
 ~~~
 
-## Mappings for Montgomery curves
+## Mappings for Montgomery curves {#montgomery}
 
-### Elligator 2 Method {#elligator2}
+The mapping defined in {{elligator2}} implements Elligator 2 {{BHKL13}} for
+curves defined by the Weierstrass equation y^2 = x^3 + A * x^2 + B * x,
+where A * B * (A^2 - 4 * B) != 0 and A^2 - 4 * B is non-square in F.
```

Comment:
I am not sure where this restriction comes from 
```
A^2 - 4 * B is non-square in F
```

Reply:
It comes from Section 5.2 of the Elligator paper. This stipulation is necessary to ensure

    v^3 + A * v^2 + B * v != 0

which handles one exceptional case.

Note that unlike Section 5.2, we do *not* stipulate that q = 1 mod 4, which means that it is possible for

    1 + u * r^2 == 0

This is the exceptional case for the mapping, and it is handled explicitly.

Reply:
Yes, I realized its purpose during the last part of my review, but I didn't remove this comment.